app/views/microposts/_form.html.erb

```diff
@@ -2,24 +2,20 @@
 <%= form_with(model: item, local: false) do |form| %>
   <section>
     <div>
-
       <div>
         <div>
+          <%= form.label :content %>
+          <%= form.text_area :content %>
+          <%= form.text_field :published_on, class: 'flatpickr', required: true %>
+          <input type="hidden" name="id" value=<%=session[:user_id]%>><br>
```

Comment:
@HrisikeshSinghaChowdhury why do we need this hidden field?

Reply:
As this partial is also called during edit ....so params will require id of the microposts...but we are displaying it at the same time